common/src/leap/soledad/common/tests/util.py

```diff
@@ -362,8 +367,14 @@ def start(self):
 %s""" % (
                     self.process.returncode, got_stdout, got_stderr))
             time.sleep(0.01)
+            if (time.time() - boot_time) > self.BOOT_TIMEOUT_SECONDS:
+                self.stop()
+                raise Exception("Timeout starting couch")
```

Comment:
What are the reasons that would cause a timeout? Could we expect this exception somewhere and try to run again?